packages/core/src/Components/Link/Internal/Types/RouterLink.ts

```
@@ -1,7 +1,7 @@
-import type { Navigation, To }  from '@/Components/Link';
-import type { RouterLinkProps } from 'vue-router';
+import type { Navigation, WithRequiredTo } from '@/Components/Link';
+import type { RouterLinkProps }            from 'vue-router';
```

ESLint

[error] Expected "vue-router" (type) to come before "@/Components/Link" (internal-type).

(perfectionist/sort-imports)